.eslintignore

```diff
@@ -5,3 +5,4 @@ build
 typechain-types
 .eslintrc.js
 commitlint.config.js
+__benchmarks__
```

Comment:
No, you don't need to ignore the whole benchmarks folder just add it to exceptions for rule import/no-extraneous-dependencies in root eslint config.

Reply:
Just use this rule
```
"import/no-extraneous-dependencies": [
      "error",
      {
        devDependencies: ["**/*.test.ts", "**/__benchmarks__/*.ts"],
      },
    ],
```

Reply:
thanks